point-collaborators/resolve-governance-tasks/inactive-workspaces.md

```diff
@@ -199,12 +199,17 @@ After selecting the Archive option for a workspace, it opens the Archive action
 
 {% hint style="warning" %}
 **Please note!**  
-This action can be executed only on workspaces that are not archived or deleted.
+This action can be executed only on workspaces that are not archived or deleted. 
 {% endhint %}
 
 {% hint style="info" %}
 **Hint!**  
-As described in the introduction, you can find the archived workspaces by switching to the Archived view on the overview screens.
+As described in the introduction, you can find the archived workspaces by switching to the **Archived view on the overview screens**.
+
+The Archive Status column shows whether a workspace was archived through Syskit Point or through Microsoft 365. **If a workspace is archived through Microsoft 365, it cannot be restored through Syskit Point**. 
```

Comment:
isto komentar da su oba archivea napravljena/pokrenuta kroz syskit point, pa raditi distinciju između in-place i m365 archival metoda. isto tako onda ova napomena ide - ako je in-place archival, ide i takav restore